Add w3f registrar guide (#1175)
w3f/polkadot-wiki: docs/learn-registrar.md

---
id: learn-registrar
title: Using W3F Registrar
sidebar_label: How to use W3F Registrar
---

> Having an on-chain identity would be a good way to build up a reputation and let the community know
> who you are if you are running a validator or being a councilor. Web3 Foundation is providing

Reviewer: ```suggestion
An on-chain identity is a good way to build up your reputation and let the community know
more about you if you plan on running a validator or being a councilor. Web3 Foundation is providing
```

> registrar service in the Polkadot network for free, so if you want your identity that is being

Reviewer: ```suggestion
registrar service in the Polkadot network for free, so if you want your identity to be
```

verified, follow the steps below to start. In case you would like to learn the basics of identity,
go [here](learn-identity).

The whole process of doing the verification is done by the bot. You will have an hour to complete
the verification. If you are not able to finish it within an hour, the judgment will be issued as
"Erroneous". Then you would have to call `clearIdentity` transaction first and redo everything
again.

> > Note: The registrar bot will not ask you to send any DOT!

Reviewer 2: "...  to send any DOT, **and never expose your private keys to anyone!**"

If you have provided `display name`, `email`, `twitter`, or `riot name` when setting an on-chain
identity, these will be required to verify one by one by signing a challenge message. Just be aware

> of the `display name` cannot be too similar to others that have verified already.

Reviewer 2: Also note that when you set `legal_name` the request is immediately rejected ("erroneous"). We do not support KYC at the time. Additional, it might be worth mentioning (as a clarification) that the user can set which ever field he/she wants (out of `display_name`, `email`, `twitter` and/or `riot_name`).

Free feel to join the [Polkadot's community](community#polkadot) to ask questions if there is
anything unclear.

## Setting an On-chain Identity

> Go to [Accounts](<(https://polkadot.js.org/apps/#/accounts)>) page in the PolkadotJS. The easiest

Reviewer: ```suggestion
Go to [Accounts](<(https://polkadot.js.org/apps/#/accounts)>) page in Polkadot-JS Apps. The easiest
```

> way to add the built-in fields is to click the gear icon next to one's account and select "Set

Reviewer: it's more of a "vertical three dots" icon not a gear icon

## Email Verification

Next, you should receive an email called "W3F Registrar Verification Service". Below is an example
for reference.

![registrar](assets/registrar/9.jpg)

You would do what you did in the above again. Copy the "Challenge" data and go to
[Sign and Verify](https://polkadot.js.org/apps/#/signing) under the Developer tab in the PolkadotJS
and select your account, paste the "Challenge" data to the "sign the following data" field and click
"Sign message"

![registrar](assets/registrar/10.jpg)

And reply with your signed data only in the email. Then click "Send".

> Note: Do not add anything in the email except the signed data.

![registrar](assets/registrar/11.jpg)

Wait for 1 to 2 minutes. You would receive another email that shows your email has been verified
successfully.

## Twitter Verification

Lastly, if you have provided Twitter handle, you would have to follow
[@w3f_registrar](https://twitter.com/w3f_registrar) first.

![registrar](assets/registrar/12.jpg)

And then wait for 1 to 2 minutes, the Twitter bot would send a challenge message to you.

![registrar](assets/registrar/13.jpg)

> Then you would do the same thing as the previous step to go to the

Reviewer 2: Is it good practice to explain the "Sign and Verify" process multiple times?

Author: Thanks. just updated ;)
